html/admin/assets.twig

```diff
@@ -299,7 +302,7 @@
                                                                                             {%  if thisasset.assignment and searchResults.PROJECT.ID != thisasset.assignment[0].projects_id %}
                                                                                                 <button title="Assigned to {{ thisasset.assignment[0]['projects_name'] }}" class="btn btn-sm btn-success" disabled><i class="fa fa-shopping-basket"></i></button>
                                                                                             {% endif %}
-                                                                                            <button title="Add to project" {% if thisasset.assignment %} style="display:none;"{% else %}{% set someassignable = true %}{% endif %} class="btn btn-sm btn-success addToBasketAssetButton" data-assetid="{{ thisasset['assets_id'] }}"><i class="fa fa-shopping-basket"></i></button>
```

Comment:
I've removed other references to this variable https://github.com/adam-rms/adam-rms/commit/f270b80917f0268ff216ce257057fd54223e2465 - could you double check what I've done?

Reply:
LGTM!